controllers/usernamespace/controller.go

```diff
@@ -600,6 +609,47 @@ func (r *CheUserNamespaceReconciler) reconcileNodeSelectorAndTolerations(ctx con
 	return r.client.Update(ctx, ns)
 }
 
+func (r *CheUserNamespaceReconciler) reconcileSCCPrivileges(username string, targetNs string, checluster *chev2.CheCluster, deployContext *chetypes.DeployContext) error {
+	if !checluster.IsOpenShiftSecurityContextConstraintSet() {
+		return nil
+	}
+
+	if username == "" {
+		return fmt.Errorf("unknow user for %s namespace", targetNs)
+	}
+
+	userSccClusterRoleBinding := &rbacv1.ClusterRoleBinding{}
+	if exists, err := deploy.Get(
+		deployContext,
+		types.NamespacedName{Name: containerbuild.GetUserSccRbacResourcesName()},
+		userSccClusterRoleBinding,
+	); !exists {
+		return err
+	}
+
+	for _, subject := range userSccClusterRoleBinding.Subjects {
+		if subject.Name == username {
+			return nil
+		}
+	}
```

Comment:
Is it required to grant `use` permissions cluster-wide? I believe this can be granted by a local rolebinding instead (though this may be harder to manage)

Reply:
Agreed. I will redo to use a rolebinding instead.

Reply:
> though this may be harder to manage

Any pitfalls do you expect?

Reply:
Not significantly, especially if this process is handled on a per-user basis as it appears to currently be. The main difficulty would be managing N roles/rolebindings for N users rather than one clusterrole/clusterrolebinding.